### DISCUSSION TITLE
How to reduce the width of empty column

### DISCUSSION THREAD

**Discussion by syvshc** [lvjr/tabularray#128] [Q&A]
I want to write a table like this 

![image](https://user-images.githubusercontent.com/38098591/136814131-7a7fd175-0378-436b-a30f-b7c77a034d53.png)

and I wrote these code:

```latex
\documentclass{article}
\usepackage{tabularray}
\begin{document}
  \begin{tblr}{
    colspec   = {cc},
    hline{2}  = {1}{solid},
    hline{4},
    vline{1}  = {2-3}{solid}
  }
   & 1. & p  \\
   & 2. & p  \\
   & 3. & p  \\
   & 4. & q  
  \end{tblr}
\end{document}
```

I got this

![image](https://user-images.githubusercontent.com/38098591/136814853-25eebf10-6c88-4ec8-b6a8-1ef522a39e3d.png)

I left the first column empty for an `hline`. However the width of first column is a little big, and `column{1} = {wd=0pt}`  doesn't work. 

Is there any way to reach my demand?

**Comment by lvjr** [comment]
Every column has default `colsep=6pt`. You may reduce it, for example, as `column{1} = {colsep=2pt}`.

**Reply by syvshc** [reply]
Thanks for your reply, I ignored `colsep` key before
